membrane_curvature/core.py

```diff
@@ -18,6 +18,7 @@
 
 sys.path.append('lib/')
```

Comment:
Can you see if you can remove this line? It shouldn't be necessary; if something breaks, you may need to `from .lib import my_func` before calling `my_func`.